python/ql/lib/semmle/python/dataflow/new/internal/FlowSummaryImplSpecific.qll

```diff
@@ -128,13 +128,33 @@ SummaryComponent interpretComponentSpecific(AccessPathToken c) {
   )
 }
 
-/** Gets the textual representation of a summary component in the format used for flow summaries. */
-string getComponentSpecific(SummaryComponent sc) {
-  sc = TContentSummaryComponent(any(ListElementContent c)) and
-  result = "ListElement"
+private string getContentSpecific(Content cs) {
```

Comment:
Perhaps just inline this into `getMadRepresentationSpecific`?

Reply:
If it is acceptable, I would prefer to keep as is, such that the structure is as similar as possible compared to the other languages.